src/test/java/com/kaltura/client/test/tests/servicesTests/transactionHistoryTests/ListTests.java

```diff
@@ -0,0 +1,119 @@
+package com.kaltura.client.test.tests.servicesTests.transactionHistoryTests;
+
+import com.kaltura.client.enums.EntityReferenceBy;
+import com.kaltura.client.enums.TransactionType;
+import com.kaltura.client.services.EntitlementService;
+import com.kaltura.client.services.EntitlementService.ListEntitlementBuilder;
+import com.kaltura.client.services.TransactionHistoryService;
+import com.kaltura.client.services.TransactionHistoryService.ListTransactionHistoryBuilder;
+import com.kaltura.client.test.tests.BaseTest;
+import com.kaltura.client.test.utils.HouseholdUtils;
+import com.kaltura.client.test.utils.PurchaseUtils;
+import com.kaltura.client.types.*;
+import com.kaltura.client.utils.response.base.Response;
+import io.qameta.allure.Severity;
+import io.qameta.allure.SeverityLevel;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ListTests extends BaseTest{
+
+    private EntitlementFilter entitlementPpvFilter;
+    private TransactionHistoryFilter transactionHistoryFilter;
+    private Response<ListResponse<Entitlement>> listEntitlementServiceResponse;
+    private Response<ListResponse<BillingTransaction>> listBillingTransactionResponse;
+    private int numberOfDevicesInHousehold = 2;
+    private int numberOfUsersInHousehold = 2;
+
+    @BeforeClass
+    public void beforeClass(){
+        entitlementPpvFilter = new EntitlementFilter();
+        entitlementPpvFilter.setEntityReferenceEqual(EntityReferenceBy.HOUSEHOLD);  //Show entitlements per household
+        entitlementPpvFilter.setIsExpiredEqual(false);  //Don't show expired assets
+        transactionHistoryFilter = new TransactionHistoryFilter();
+        transactionHistoryFilter.setStartDateGreaterThanOrEqual((int)((System.currentTimeMillis()/1000)-86400));    //Start date from yesterday (86400 sec = 24 hours)
```

Comment:
please use variables instead of "magic numbers" like 1000 and 86400. Even when you use comments. It is better for future support